jest.config.js

```diff
@@ -6,5 +6,5 @@ module.exports = {
         "^.+\\.tsx?$": "ts-jest"
     },
     "verbose": true,
-    "forceExit": true
+    "testEnvironment": "node"
```

Comment:
is this how the tests work with you? I added this because of a unclosed session (not very well, but it worked ;))

Reply:
I closed everything (db connection, mqtt broker) when closing the server, so it shutdowns correctly now.